src/Datagrid/Pager.php

```diff
@@ -492,6 +492,7 @@ public function next()
 
         --$this->resultsCounter;
 
+        // NEXT_MAJOR: remove this line. Iterator::next(): void
```

Comment:
Are you sure this next line has to be removed? Do you mean keep calling next but not returning anything or complete removal? AFAIK complete remove the line will cause this method do nothing or am I wrong?

Reply:
Since there is the signature `Iterator::next(): void`, I think only the return in `next` and `rewind` should be remove indeed. The NEXT_MAJOR comment should be fixed.